site/content/en/docs/tasks/security/jwt-authentication.md

```diff
@@ -91,7 +91,7 @@ A `401` HTTP response code should be returned.
 Get the JWT used for testing request authentication:
 
 ```shell
-TOKEN=$(curl https://raw.githubusercontent.com/envoyproxy/gateway/main/examples/kubernetes/jwt/test.jwt -s) && echo "$TOKEN" | cut -d '.' -f2 - | base64 --decode -
```

Comment:
did you test the new command on linux?

Reply:
Yes, this command works on Linux. In fact, the existing command only works on Linux, not on mac. The changed command works both.

Reply:
thanks for clarifying.